smash/core/simulation/run/run.py

```diff
@@ -192,7 +198,13 @@ def _forward_run(
         fret[key] = value
 
     ret = {**fret, **pyret}
+
     if ret:
+        if "internal_fluxes" in ret:
+            ret["internal_fluxes"] = transform_internal_fluxes_to_dict(
+                ret["internal_fluxes"], STRUCTURE_RR_INTERNAL_FLUXES, model.setup.structure
+            )
```

Comment:
Ok pour renvoyer un dictionnaire. Je trouve que ca fait bcp de faire une fonction spécifique dans un autre fichier pour ca
```suggestion
        if "internal_fluxes" in ret:
            ret["internal_fluxes"] = {key: ret["internal_fluxes"][..., i] for i, key in enumerate(STRUCTURE_RR_INTERNAL_FLUXES[model.setup.structure])}
```

Reply:
J'étais partie pour surcharger le tableau fortran via une classe python. C'est pourquoi j'ai commencé ailleurs.
Je me suis rendue compte qu'une fonction étais plus adéquate, et j'ai gardé le fichier.